### ISSUE TITLE
Write "Classical statistical mechanics" section

### ISSUE THREAD

**Issue by JIMonroe** [MobleyLab/basic_simulation_training#18]
This is an important one, but also something we don't want to spend too much time summarizing, as there are already many great resources out there.  Mainly the task will be to write a very concise overview with an eye towards knowledge necessary for understanding MD simulations, then provide an excellent, specific references list.

**Comment by davidlmobley** [comment]
I'm not actually too sure how to deal with this one; maybe we should just try and write a really brief amount on some of the key issues and why they are important and link out a lot.

I'll write Dan Zuckerman and see if he has any suggestions.

Actually, better yet, I'll tag him here.

@dmzuckerman we are working on getting our "simulation basics" best practices document together for submission. Basically it's sort of a "what you had better make sure you know before going too far" kind of document which we intend to be heavy with references to good sources (rather than turning it into a book). Some topics we can address -- what makes a good barostat or thermostat, what happens if you pick a bad one -- and others we have to really just describe what people should learn about (e.g. they need to know classical mechanics and we're not going to teach it to them). 

One section that's tough is the one on classical statistical mechanics.  We can't teach a course on it in an article, so to some extent it has to be a list of what people should know and where to find out about it. But we may be able to put in a bit of info anyway. Alternatively, maybe there are good concise introductions to the basics which are already out there. Thoughts?

Here's what we had put in our outline about what would perhaps go here:
```latex
\item Ensembles, distribution functions for different ensembles. Equivalence of ensembles
\item What equilibrium means and difference between equilibrium and
    non-equilibrium.
    For instance, what is usually called an ``equilibrium trajectory'' generally will not embody a good sample of the equilibrium ensemble due to insufficient sampling.  
        On the other hand, truly non-equilibrium conditions such as driven transitions and relaxation are fundamentally different. 
        Note that relaxation can occur to the equilibrium ensemble or a non-equilibrium condition (e.g., steady state).
\item Clarify differences between nonequilibrium ensembles: driven
    nonequilibrium steady-state, systems driven out of equilibrium by a time-dependent field, systems initially out of equilibrium but relaxing
        to equilibrium
\item Time averages and ensemble averages
\item Fluctuations
\item Correlation functions
```

**Comment by dmzuckerman** [comment]
I think I wrote a lot of that brief outline!  I would consider lending a hand with this - i.e., trying to write about one page on essential concepts with links out to references.  The problem is that since I have devoted so much of my 'free time' to the subject, I will mostly self-reference my book and perhaps blog.  I would welcome additional references put in, of course.  So think about whether you want an additional author and under these conditions.

**Comment by davidlmobley** [comment]
@dmzuckerman - I really like your book, and others speak highly of it as well. I'd be totally fine with accepting your contribution on those conditions. Plan on it?? 

I'll "assign" the issue to you. When you edit, please add yourself as an author. :)

**Comment by davidlmobley** [comment]
(Actually I have to invite you to the repo first, which I'll do now.)

**Comment by davidlmobley** [comment]
Closed via #28
